--- ltris-1.0.15/src/chart.c.orig	2009-12-25 12:38:12.000000000 +0300
+++ ltris-1.0.15/src/chart.c	2010-12-29 17:44:42.000000000 +0300
@@ -189,15 +189,15 @@
 	if ( charts ) list_delete( charts ); charts = 0;
 	charts = list_create( LIST_AUTO_DELETE, chart_set_delete );
     /* load highscore */
-    if ( !chart_load_from_path( HI_DIR ) ) {
-        fprintf( stderr, "Unable to access highscore chart in '%s'.\n", HI_DIR );
-        fprintf( stderr, "Trying to use config directory '%s'.\n", config.dir_name );
+//    if ( !chart_load_from_path( HI_DIR ) ) {
+//        fprintf( stderr, "Unable to access highscore chart in '%s'.\n", HI_DIR );
+//        fprintf( stderr, "Trying to use config directory '%s'.\n", config.dir_name );
         if ( !chart_load_from_path( config.dir_name ) ) {
             fprintf( stderr, "Unable to access highscore chart in config directory... won't be "
                              "able to save any results. Sorry.\n" );
             return;
         }
-    }
+//    }
     printf( "Saving highscore chart in: %s\n", chart_path );
     /* load resources */
     cfont = load_fixed_font( "f_small_yellow.bmp", 32, 96, 8 );
