--- a/util/bash-completion.d/grub-completion.bash.in.orig	2011-06-28 18:29:25.000000000 +0400
+++ a/util/bash-completion.d/grub-completion.bash.in	2011-06-28 18:42:17.610283552 +0400
@@ -402,7 +402,7 @@
 #
 # grub-mkpasswd-pbkdf2
 #
-_grub_mkpasswd-pbkdf2 () {
+_grub_mkpasswd_pbkdf2 () {
     local cur
 
     COMPREPLY=()
@@ -417,7 +417,7 @@
 }
 __grub_mkpasswd_pbkdf2_program=$( echo grub-mkpasswd-pbkdf2 | sed "s,x,x," )
 have ${__grub_mkpasswd_pbkdf2_program} && \
- complete -F _grub_mkpasswd-pbkdf2 -o filenames ${__grub_mkpasswd_pbkdf2_program}
+ complete -F _grub_mkpasswd_pbkdf2 -o filenames ${__grub_mkpasswd_pbkdf2_program}
 unset __grub_mkpasswd_pbkdf2_program
 
 
@@ -462,7 +462,7 @@
 #
 # grub-script-check
 #
-_grub_script-check () {
+_grub_script_check () {
     local cur
 
     COMPREPLY=()
@@ -477,7 +477,7 @@
 }
 __grub_script_check_program=$( echo grub-script-check | sed "s,x,x," )
 have ${__grub_script_check_program} && \
- complete -F _grub_script-check -o filenames ${__grub_script_check_program}
+ complete -F _grub_script_check -o filenames ${__grub_script_check_program}
 
 
 # Local variables:
