--- laptop-mode-tools_1.60/install.sh	2011-10-14 11:40:08.000000000 +0400
+++ laptop-mode-tools_1.60-agilia/install.sh	2011-12-25 01:35:19.711762592 +0400
@@ -12,6 +12,7 @@
 # installation of init scripts.
 # MAN_D = directory in which the manual pages are to be installed. This should
 # be the base directory, e.g. /usr/local/man.
+# LIB_SUFFIX = destination /usr/lib directory (depends on host architecture)
 # INSTALL = command line for install program. By default, this is set to
 # "install -o root -g root".
 #
@@ -27,6 +28,8 @@
 # and "auto" means: autodetect if this power management method is supported.
 #
 
+[ -z "$LIB_SUFFIX" ] && LIB_SUFFIX=""
+
 [ -z "$MAN_D" ] && MAN_D="/usr/man"
 
 if [ -z "$ACPI" ] ; then
@@ -182,15 +185,15 @@
 
 # Install pm-utils hook only if pm-utils config dir is present, or force the
 # install if we have a DESTDIR.
-if [ -n "$DESTDIR" -o -d "$DESTDIR/usr/lib/pm-utils/sleep.d" ] ; then
-	if ( ! $INSTALL -D -m 755 usr/lib/pm-utils/sleep.d/01laptop-mode "$DESTDIR/usr/lib/pm-utils/sleep.d/01laptop-mode" ) ; then
-		echo "$0: Failed to install 01-laptop-mode into /usr/lib/pm-utils/sleep.d. Installation failed."
+if [ -n "$DESTDIR" -o -d "$DESTDIR/usr/lib${LIB_SUFFIX}/pm-utils/sleep.d" ] ; then
+	if ( ! $INSTALL -D -m 755 usr/lib/pm-utils/sleep.d/01laptop-mode "$DESTDIR/usr/lib${LIB_SUFFIX}/pm-utils/sleep.d/01laptop-mode" ) ; then
+		echo "$0: Failed to install 01-laptop-mode into /usr/lib${LIB_SUFFIX}/pm-utils/sleep.d. Installation failed."
 		exit 36
 	fi
 fi
 
-if [ -f "$DESTDIR/usr/lib/pm-utils/sleep.d/99laptop-mode" ]; then
-	rm -f $DESTDIR/usr/lib/pm-utils/sleep.d/99laptop-mode;
+if [ -f "$DESTDIR/usr/lib${LIB_SUFFIX}/pm-utils/sleep.d/99laptop-mode" ]; then
+	rm -f $DESTDIR/usr/lib${LIB_SUFFIX}/pm-utils/sleep.d/99laptop-mode;
 fi
 
 # udev rule
