diff --git a/Apackager.desktop b/Apackager.desktop
index 349d40d..cd8ba70 100644
--- a/Apackager.desktop
+++ b/Apackager.desktop
@@ -9,13 +9,12 @@ GenericName=Package manager Apackager
 GenericName[ru]=Менеджер пакетов Apackager
 Icon=mpkg-apackager
 MimeType=
-Name=MPKG Package Apackager (mpkg)
+Name=Apackager: package manager GUI
 Name[ru]=Установка/удаление пакетов
 Name[en]=Install/remove packages
 Path=
 StartupNotify=true
 Terminal=false
-TerminalOptions
 Type=Application
 X-DCOP-ServiceType=
 X-KDE-SubstituteUID=false
