diff -uNrp Blue_Grass.orig/blue_grass.edc Blue_Grass/blue_grass.edc
--- Blue_Grass.orig/blue_grass.edc	2010-10-15 01:05:43.624279000 +0800
+++ Blue_Grass/blue_grass.edc	2010-10-15 21:35:17.691000002 +0800
@@ -6,10 +6,11 @@ data {
 }
 
 fonts {
-   font: "Vera.ttf"   "Edje-Vera";
-   font: "VeraBd.ttf" "Edje-Vera-Bold";
+   font: "DejaVuSans.ttf"   "Edje-Vera";
+   font: "DejaVuSans-Bold.ttf" "Edje-Vera-Bold";
 }
-
+ 
+#if 0
 spectra {
   spectrum {
     name: "black_to_trans";
@@ -23,6 +24,7 @@ spectra {
 	color: 255 255 255 127 1;
      }
 }
+#endif
 
 color_classes {
    color_class {
diff -uNrp Blue_Grass.orig/default_fileman.edc Blue_Grass/default_fileman.edc
--- Blue_Grass.orig/default_fileman.edc	2010-10-15 01:05:43.627279000 +0800
+++ Blue_Grass/default_fileman.edc	2010-10-15 21:36:26.749000002 +0800
@@ -2290,6 +2290,7 @@ group 
 		  color: 255 255 255 128;
 	       }
 	  }
+#if 0
 	part 
 	  {
 	     name: "grad";
@@ -2306,6 +2307,7 @@ group 
 		  gradient.rel2.offset: -1 0;
 	       }
 	  }
+#endif
      }
 }
 
