diff -uNrp cerium.orig/cerium.edc cerium/cerium.edc
--- cerium.orig/cerium.edc	2010-10-15 01:06:07.604279000 +0800
+++ cerium/cerium.edc	2010-10-15 22:00:31.722000003 +0800
@@ -6,10 +6,11 @@ data {
 }
 
 fonts {
-   font: "Vera.ttf"   "Edje-Vera";
-   font: "VeraBd.ttf" "Edje-Vera-Bold";
+   font: "DejaVuSans.ttf"   "Edje-Vera";
+   font: "DejaVuSans-Bold.ttf" "Edje-Vera-Bold";
 }
 
+#if 0
 spectra {
   spectrum {
     name: "black_to_trans";
@@ -23,6 +24,7 @@ spectra {
 	color: 255 255 255 127 1;
      }
 }
+#endif
 
 color_classes {
    color_class {
diff -uNrp cerium.orig/default_fileman.edc cerium/default_fileman.edc
--- cerium.orig/default_fileman.edc	2010-10-15 01:06:07.607279000 +0800
+++ cerium/default_fileman.edc	2010-10-15 22:01:33.477000002 +0800
@@ -2290,6 +2290,7 @@ group 
 		  color: 255 255 255 128;
 	       }
 	  }
+#if 0
 	part 
 	  {
 	     name: "grad";
@@ -2306,6 +2307,7 @@ group 
 		  gradient.rel2.offset: -1 0;
 	       }
 	  }
+#endif
      }
 }
 
