diff -uNrp grunge.orig/default_icons.edc grunge/default_icons.edc
--- grunge.orig/default_icons.edc	2010-10-15 01:09:16.625279000 +0800
+++ grunge/default_icons.edc	2010-10-16 02:27:41.131000003 +0800
@@ -1819,6 +1819,7 @@ group {
    name: "e/icons/enlightenment/gradient_h";
    max:  24 24;
    parts {
+#if 0
       part {
 	 name:          "bg";
          type: GRADIENT;
@@ -1841,6 +1842,7 @@ group {
             }
 	 }
       }
+#endif
       part {
 	 name:          "screen";
 	 mouse_events:  0;
@@ -1861,6 +1863,7 @@ group {
    name: "e/icons/enlightenment/gradient_v";
    max:  24 24;
    parts {
+#if 0
       part {
 	 name:          "bg";
          type: GRADIENT;
@@ -1879,6 +1882,7 @@ group {
             }
 	 }
       }
+#endif
       part {
 	 name:          "screen";
 	 mouse_events:  0;
@@ -1899,6 +1903,7 @@ group {
    name: "e/icons/enlightenment/gradient_du";
    max:  24 24;
    parts {
+#if 0
       part {
 	 name:          "bg";
          type: GRADIENT;
@@ -1921,6 +1926,7 @@ group {
             }
 	 }
       }
+#endif
       part {
 	 name:          "screen";
 	 mouse_events:  0;
@@ -1941,6 +1947,7 @@ group {
    name: "e/icons/enlightenment/gradient_dd";
    max:  24 24;
    parts {
+#if 0
       part {
 	 name:          "bg";
          type: GRADIENT;
@@ -1963,6 +1970,7 @@ group {
             }
 	 }
       }
+#endif
       part {
 	 name:          "screen";
 	 mouse_events:  0;
@@ -1983,6 +1991,7 @@ group {
    name: "e/icons/enlightenment/gradient_rad";
    max:  24 24;
    parts {
+#if 0
       part {
 	 name:          "bg";
          type: GRADIENT;
@@ -2005,6 +2014,7 @@ group {
             }
 	 }
       }
+#endif
       part {
 	 name:          "screen";
 	 mouse_events:  0;
diff -uNrp grunge.orig/grunge.edc grunge/grunge.edc
--- grunge.orig/grunge.edc	2010-10-15 01:09:16.623279000 +0800
+++ grunge/grunge.edc	2010-10-16 02:25:37.354000002 +0800
@@ -7,9 +7,9 @@ data {
 
 
 fonts {
-   font: "Vera.ttf"   "Edje-Vera";
-   font: "VeraBd.ttf" "Edje-Vera-Bold";
-   font: "SoulMission.ttf" "SoulMission";
+   font: "DejaVuSans.ttf"   "Edje-Vera";
+   font: "DejaVuSans-Bold.ttf" "Edje-Vera-Bold";
+   font: "DejaVuSans.ttf" "SoulMission";
 }
 
 color_classes {
@@ -23,6 +23,7 @@ color_classes {
    }
 }
 
+#if 0
 spectra {
   spectrum {
     name: "black_to_trans";
@@ -30,6 +31,7 @@ spectra {
     color: 0 0 0 0 1;
   }
 }
+#endif
 
 
 collections {
