--- ./cmake/FindIDNA.cmake	2012-02-21 06:55:51.000000000 +1100
+++ ./FindIDNA.cmake	2012-02-24 04:52:43.633572581 +1100
@@ -1,9 +1,9 @@
 # - Try to find GNU IDN library and headers
 # Once done, this will define
 #
-#  IDNA_FOUND - system has IDNA
-#  IDNA_INCLUDE_DIR - the IDNA include directories (<idna.h>)
-#  IDNA_LIBRARIES - link these to use IDNA (idna_to_ascii_8z)
+# IDNA_FOUND - system has IDNA
+# IDNA_INCLUDE_DIR - the IDNA include directories (<idna.h>)
+# IDNA_LIBRARIES - link these to use IDNA (idna_to_ascii_8z)
 
 if (IDNA_INCLUDE_DIR AND IDNA_LIBRARIES)
   set(IDNA_FIND_QUIETLY TRUE)
@@ -36,5 +36,5 @@
 
 # Lastly make it so that the IDNA_LIBRARIES and IDNA_INCLUDE_DIR variables
 # only show up under the advanced options in the gui cmake applications.
-MARK_AS_ADVANCED( IDNA_LIBRARIES IDNA_INCLUDE_DIR )
+MARK_AS_ADVANCED( IDNA_LIBRARIES IDNA_INCLUDE_DIR IDNA_LIBRARY)
 
