--- ./cmake/FindXattr.cmake	2012-02-21 06:55:51.000000000 +1100
+++ ./FindXattr.cmake	2012-02-24 04:56:15.257573934 +1100
@@ -1,9 +1,9 @@
 # - Try to find libattr library and headers
 # Once done, this will define
 #
-#  XATTR_FOUND - system has libattr
-#  XATTR_INCLUDE_DIRS - the libattr include directories
-#  XATTR_LIBRARIES - link these to use libattr
+# XATTR_FOUND - system has libattr
+# XATTR_INCLUDE_DIRS - the libattr include directories
+# XATTR_LIBRARIES - link these to use libattr
  
 FIND_PATH(XATTR_INCLUDE xattr.h
   ${XATTR_PREFIX}/include/attr
@@ -26,3 +26,5 @@
   SET(XATTR_FOUND FALSE)
   SET(XATTR_LIBRARIES "")
 ENDIF(XATTR_INCLUDE AND XATTR_LIB)
+
+MARK_AS_ADVANCED( XATTR_LIB XATTR_INCLUDE )
