--- eiskaltdcpp-daemon/xmlrpcserver.h	2012-02-21 06:55:51.000000000 +1100
+++ xmlrpcserver.h	2012-02-24 05:09:20.890578960 +1100
@@ -349,8 +349,10 @@
         string const shub(paramList.getString(0));
         string const snick(paramList.getString(1));
         paramList.verifyEnd(2);
-        string tmp = ServerThread::getInstance()->getFileList_client(shub, snick, false);
-        *retvalP = xmlrpc_c::value_string(tmp);
+        if (ServerThread::getInstance()->getFileList(shub, snick, false))
+            *retvalP = xmlrpc_c::value_int(0);
+        else
+            *retvalP = xmlrpc_c::value_int(1);
     }
 };
 
