diff -uNr comm-1.9.2/calendar/sunbird/app/profile/mimeTypes.rdf comm-1.9.2.new/calendar/sunbird/app/profile/mimeTypes.rdf
--- comm-1.9.2/calendar/sunbird/app/profile/mimeTypes.rdf	2010-06-08 13:04:17.000000000 +0400
+++ comm-1.9.2.new/calendar/sunbird/app/profile/mimeTypes.rdf	2010-06-17 17:12:01.000000000 +0400
@@ -86,4 +86,16 @@
                    NC:value="text/calendar"
                    NC:editable="true">
   </RDF:Description> 
+<RDF:Description RDF:about="urn:scheme:file"
+NC:value="file">
+<NC:handlerProp RDF:resource="urn:scheme:handler:file"/>
+</RDF:Description>
+<RDF:Description RDF:about="urn:scheme:handler:file"
+NC:alwaysAsk="false">
+<NC:possibleApplication RDF:resource="urn:scheme:externalApplication:file"/>
+</RDF:Description>
+<RDF:Description RDF:about="urn:scheme:externalApplication:file"
+NC:prettyName="AnyName"
+NC:path="/usr/bin/xdg-open"/>
+
 </RDF:RDF>
diff -uNr comm-1.9.2/mail/app/profile/mimeTypes.rdf comm-1.9.2.new/mail/app/profile/mimeTypes.rdf
--- comm-1.9.2/mail/app/profile/mimeTypes.rdf	2010-06-08 13:04:17.000000000 +0400
+++ comm-1.9.2.new/mail/app/profile/mimeTypes.rdf	2010-06-17 17:20:03.000000000 +0400
@@ -79,4 +79,15 @@
       </RDF:Seq>
     </NC:MIME-types> 
   </RDF:Description> 
+<RDF:Description RDF:about="urn:scheme:file"
+NC:value="file">
+<NC:handlerProp RDF:resource="urn:scheme:handler:file"/>
+</RDF:Description>
+<RDF:Description RDF:about="urn:scheme:handler:file"
+NC:alwaysAsk="false">
+<NC:possibleApplication RDF:resource="urn:scheme:externalApplication:file"/>
+</RDF:Description>
+<RDF:Description RDF:about="urn:scheme:externalApplication:file"
+NC:prettyName="AnyName"
+NC:path="/usr/bin/xdg-open"/>
 </RDF:RDF>
diff -uNr comm-1.9.2/mozilla/browser/locales/generic/profile/mimeTypes.rdf comm-1.9.2.new/mozilla/browser/locales/generic/profile/mimeTypes.rdf
--- comm-1.9.2/mozilla/browser/locales/generic/profile/mimeTypes.rdf	2010-06-08 13:07:04.000000000 +0400
+++ comm-1.9.2.new/mozilla/browser/locales/generic/profile/mimeTypes.rdf	2010-06-17 17:25:45.000000000 +0400
@@ -1,13 +1,40 @@
 <?xml version="1.0"?>  
 
-<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
-     xmlns:NC="http://home.netscape.com/NC-rdf#"
-     xmlns:RDF="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
-
-  <Description about="urn:mimetypes"> 
-    <NC:MIME-types> 
-      <Seq about="urn:mimetypes:root"> 
-      </Seq> 
-    </NC:MIME-types> 
-  </Description> 
-</RDF>
+<RDF:RDF xmlns:NC="http://home.netscape.com/NC-rdf#"
+         xmlns:RDF="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+  <RDF:Seq RDF:about="urn:mimetypes:root">
+  </RDF:Seq>
+  <RDF:Description RDF:about="urn:mimetypes">
+    <NC:MIME-types RDF:resource="urn:mimetypes:root"/>
+  </RDF:Description>
+  <RDF:Description RDF:about="urn:scheme:mailto"
+                   NC:value="mailto">
+    <NC:handlerProp RDF:resource="urn:scheme:handler:mailto"/>
+  </RDF:Description>
+  <RDF:Description RDF:about="urn:handler:web:http://compose.mail.yahoo.com/?To=%s"
+                   NC:prettyName="Yahoo! Mail"
+                   NC:uriTemplate="http://compose.mail.yahoo.com/?To=%s" />
+  <RDF:Description RDF:about="urn:handler:web:https://mail.google.com/mail/?extsrc=mailto&amp;url=%s"
+                   NC:prettyName="GMail"
+                   NC:uriTemplate="https://mail.google.com/mail/?extsrc=mailto&amp;url=%s" />
+  <RDF:Description RDF:about="urn:scheme:handler:mailto"
+                   NC:alwaysAsk="true">
+    <NC:possibleApplication RDF:resource="urn:handler:web:http://compose.mail.yahoo.com/?To=%s"/>
+    <NC:poODssibleApplication RDF:resource="urn:handler:web:https://mail.google.com/mail/?extsrc=mailto&amp;url=%s"/>                                                                                                
+    <NC:possibleApplication RDF:resource="urn:handler:local:/usr/bin/thunderbird"/>
+  </RDF:Description>
+  <RDF:Description RDF:about="urn:handler:local:/usr/bin/thunderbird"
+                   NC:prettyName="thunderbird"
+                   NC:path="/usr/bin/thunderbird" />
+<RDF:Description RDF:about="urn:scheme:file"
+NC:value="file">
+<NC:handlerProp RDF:resource="urn:scheme:handler:file"/>
+</RDF:Description>
+<RDF:Description RDF:about="urn:scheme:handler:file"
+NC:alwaysAsk="false">
+<NC:possibleApplication RDF:resource="urn:scheme:externalApplication:file"/>
+</RDF:Description>
+<RDF:Description RDF:about="urn:scheme:externalApplication:file"
+NC:prettyName="AnyName"
+NC:path="/usr/bin/xdg-open"/>
+</RDF:RDF>
