--- src/e_mod_config.c.orig	2010-10-06 15:56:25.315000143 +0800
+++ src/e_mod_config.c	2010-10-13 11:52:11.047279622 +0800
@@ -34,7 +34,7 @@ _config_empris_module (Config_Item * ci)
 	    e_module_dir_get (empris_config->module));
   con = e_container_current_get (e_manager_current_get ());
   cfd =
-    e_config_dialog_new (con, D_ ("Empris Configuration"), "Empris",
+    e_config_dialog_new (con, D_ ("Empris Settings"), "Empris",
 			 "_e_modules_empris_config_dialog", buf, 0, v, ci);
   empris_config->config_dialog = cfd;
 }


--- src/e_mod_main.c.orig	2010-10-06 15:56:25.314000143 +0800
+++ src/e_mod_main.c	2010-10-13 11:32:08.571279619 +0800
@@ -234,24 +234,24 @@ _empris_cb_mouse_down (void *data, Evas 
    if ((ev->button == 3) && (!empris_config->menu))
      {
 	E_Menu *mn;
+	E_Menu *ma, *mg;
 	E_Menu_Item *mi;
 	int x, y, w, h;
 
-	mn = e_menu_new ();
-	e_menu_post_deactivate_callback_set (mn, _empris_menu_cb_post, inst);
-	empris_config->menu = mn;
+	ma = e_menu_new ();
+	e_menu_post_deactivate_callback_set (ma, _empris_menu_cb_post, inst);
+	empris_config->menu = ma;
 
-	mi = e_menu_item_new (mn);
-	e_menu_item_label_set (mi, D_ ("Configuration"));
+	mg = e_menu_new ();
+	
+	mi = e_menu_item_new (mg);
+	e_menu_item_label_set (mi, D_ ("Settings"));
 	e_util_menu_item_theme_icon_set(mi, "preferences-system");
 	e_menu_item_callback_set (mi, _empris_menu_cb_configure, inst);
 
-	mi = e_menu_item_new (mn);
-	e_menu_item_separator_set (mi, 1);
-
-	e_gadcon_client_util_menu_items_append (inst->gcc, mn, 0);
+	e_gadcon_client_util_menu_items_append (inst->gcc, ma, mg, 0);
 	e_gadcon_canvas_zone_geometry_get (inst->gcc->gadcon, &x, &y, &w, &h);
-	e_menu_activate_mouse (mn,
+	e_menu_activate_mouse (ma,
 			       e_util_zone_current_get (e_manager_current_get
 							()), x + ev->output.x,
 			       y + ev->output.y, 1, 1,
