--- goldendict.pro.orig	2010-01-29 15:04:22.000000000 +0300
+++ goldendict.pro	2010-01-29 15:04:30.000000000 +0300
@@ -47,7 +47,7 @@
     LIBS += -lX11 \
         -lXtst
     PREFIX = $$(PREFIX)
-    isEmpty( PREFIX ):PREFIX = /usr/local
+    isEmpty( PREFIX ):PREFIX = /usr
     DEFINES += PROGRAM_DATA_DIR=\\\"$$PREFIX/share/apps/goldendict/\\\"
     target.path = $$PREFIX/bin/
     locale.path = $$PREFIX/share/apps/goldendict/locale/
