--- libcompizconfig-0.9.5.0.orig/cmake/LibCompizConfigCommon.cmake	2011-08-22 21:34:41.345982076 +0400
+++ libcompizconfig-0.9.5.0/cmake/LibCompizConfigCommon.cmake	2011-08-22 21:35:54.451982010 +0400
@@ -122,7 +122,7 @@
 
     install (
 	TARGETS ${bname}
-	DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/lib/compizconfig/backends
+	DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/compizconfig/backends
     )
 
     if (NOT _COMPIZCONFIG_INTERNAL)
