diff -urN libgksu-2.0.12.orig/libgksu/Makefile.am libgksu-2.0.12/libgksu/Makefile.am
--- libgksu-2.0.12.orig/libgksu/Makefile.am	2014-07-01 15:05:51.997312025 +0300
+++ libgksu-2.0.12/libgksu/Makefile.am	2014-07-01 15:06:15.518301422 +0300
@@ -22,8 +22,8 @@
 pkgconfigdir = ${libdir}/pkgconfig
 pkgconfig_DATA = libgksu2.pc
 
-pkglibdir = ${libdir}/${PACKAGE}
-pkglib_PROGRAMS = gksu-run-helper
+gksulibdir = ${libdir}/${PACKAGE}
+gksulib_PROGRAMS = gksu-run-helper
 gksu_run_helper_LDFLAGS = `pkg-config --libs glib-2.0`
 gksu_run_helper_SOURCES = gksu-run-helper.c
 
