Index: ebook-tools-0.2.1/CMakeLists.txt
===================================================================
--- ebook-tools-0.2.1.orig/CMakeLists.txt	2010-02-28 18:09:46.000000000 +0100
+++ ebook-tools-0.2.1/CMakeLists.txt	2011-03-23 10:02:24.543822958 +0100
@@ -2,11 +2,14 @@ project (EBOOK-TOOLS C)
 
 cmake_minimum_required(VERSION 2.4.0)
 
+find_package(PkgConfig)
+pkg_check_modules(ZIP libzip)
 #set(CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src)
 set(LIBRARY_OUTPUT_PATH ${EBOOK-TOOLS_BINARY_DIR}/libs)
 set(EXECUTABLE_OUTPUT_PATH ${EBOOK-TOOLS_BINARY_DIR}/bin)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+set(LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)" )
 
 find_package(LibXml2 REQUIRED)
 find_package(LibZip REQUIRED)
Index: ebook-tools-0.2.1/src/libepub/CMakeLists.txt
===================================================================
--- ebook-tools-0.2.1.orig/src/libepub/CMakeLists.txt	2010-09-13 21:31:53.000000000 +0200
+++ ebook-tools-0.2.1/src/libepub/CMakeLists.txt	2011-03-23 10:08:59.695872613 +0100
@@ -1,8 +1,8 @@
-include_directories (${EBOOK-TOOLS_SOURCE_DIR}/src/libepub ${LIBXML2_INCLUDE_DIR} ${LIBZIP_INCLUDE_DIR})
+include_directories (${EBOOK-TOOLS_SOURCE_DIR}/src/libepub ${LIBXML2_INCLUDE_DIR} ${LIBZIP_INCLUDE_DIR} ${ZIP_INCLUDE_DIRS} )
 add_library (epub SHARED epub.c ocf.c opf.c linklist.c list.c)
 target_link_libraries (epub ${LIBZIP_LIBRARY} ${LIBXML2_LIBRARIES})
 
 set_target_properties (epub PROPERTIES VERSION 0.2.1 SOVERSION 0)
 
-install ( TARGETS epub RUNTIME DESTINATION bin LIBRARY DESTINATION lib ARCHIVE DESTINATION lib )
+install ( TARGETS epub RUNTIME DESTINATION bin LIBRARY DESTINATION lib${LIB_SUFFIX} ARCHIVE DESTINATION lib${LIB_SUFFIX} )
 install ( FILES epub.h epub_shared.h epub_version.h DESTINATION include )
