--- php-fpm.conf.orig	2013-06-11 01:16:01.000000000 +0400
+++ php-fpm.conf	2013-06-11 03:26:00.000000000 +0400
@@ -29,7 +29,7 @@
 ; in a local file.
 ; Note: the default prefix is /usr/var
 ; Default Value: log/php-fpm.log
-;error_log = log/php-fpm.log
+error_log = /var/log/php-fpm.log
 
 ; syslog_facility is used to specify what type of program is logging the
 ; message. This lets syslogd specify that messages from different facilities
