--- ./Makefile_orig	2010-07-29 03:52:12.000000000 +1100
+++ ./Makefile	2010-12-12 01:19:18.732000012 +1000
@@ -35,7 +35,7 @@
 SOURCES := bitops.c libnuma.c distance.c memhog.c numactl.c numademo.c \
 	numamon.c shm.c stream_lib.c stream_main.c syscall.c util.c mt.c \
 	clearcache.c test/*.c
-
+destdir := ${pkgdir}
 prefix := /usr
 libdir := ${prefix}/$(shell ./getlibdir)
 docdir := ${prefix}/share/doc
@@ -128,26 +128,26 @@
 MANPAGES := numa.3 numactl.8 numastat.8 migratepages.8 migspeed.8
 
 install: numactl migratepages migspeed numademo.c numamon memhog libnuma.so.1 numa.h numaif.h numacompat1.h numastat ${MANPAGES}
-	mkdir -p ${prefix}/bin
-	install -m 0755 numactl ${prefix}/bin
-	install -m 0755 migratepages ${prefix}/bin
-	install -m 0755 migspeed ${prefix}/bin
-	install -m 0755 numademo ${prefix}/bin
-	install -m 0755 memhog ${prefix}/bin
-	mkdir -p ${prefix}/share/man/man2 ${prefix}/share/man/man8 ${prefix}/share/man/man3
-	install -m 0644 numactl.8 ${prefix}/share/man/man8
-	install -m 0644 numa.3 ${prefix}/share/man/man3
-	( cd ${prefix}/share/man/man3 ; for i in $$(./manlinks) ; do ln -sf numa.3 $$i.3 ; done )
-	mkdir -p ${libdir}
-	install -m 0755 libnuma.so.1 ${libdir}
-	cd ${libdir} ; ln -sf libnuma.so.1 libnuma.so
-	install -m 0644 libnuma.a ${libdir}
-	mkdir -p ${prefix}/include
-	install -m 0644 numa.h numaif.h numacompat1.h ${prefix}/include
-	install -m 0755 numastat ${prefix}/bin
-	if [ -d ${docdir} ] ; then \
-		mkdir -p ${docdir}/numactl/examples ; \
-		install -m 0644 numademo.c ${docdir}/numactl/examples ; \
+	mkdir -p ${destdir}${prefix}/bin
+	install -m 0755 numactl ${destdir}${prefix}/bin
+	install -m 0755 migratepages ${destdir}${prefix}/bin
+	install -m 0755 migspeed ${destdir}${prefix}/bin
+	install -m 0755 numademo ${destdir}${prefix}/bin
+	install -m 0755 memhog ${destdir}${prefix}/bin
+	mkdir -p ${destdir}${prefix}/share/man/man2 ${destdir}${prefix}/share/man/man8 ${destdir}${prefix}/share/man/man3
+	install -m 0644 numactl.8 ${destdir}${prefix}/share/man/man8
+	install -m 0644 numa.3 ${destdir}${prefix}/share/man/man3
+	( cd ${destdir}${prefix}/share/man/man3 ; for i in $$(./manlinks) ; do ln -sf numa.3 $$i.3 ; done )
+	mkdir -p ${destdir}${libdir}
+	install -m 0755 libnuma.so.1 ${destdir}${libdir}
+	cd ${destdir}${libdir} ; ln -sf libnuma.so.1 libnuma.so
+	install -m 0644 libnuma.a ${destdir}${libdir}
+	mkdir -p ${destdir}${prefix}/include
+	install -m 0644 numa.h numaif.h numacompat1.h ${destdir}${prefix}/include
+	install -m 0755 numastat ${destdir}${prefix}/bin
+	if [ -d ${destdir}${docdir} ] ; then \
+		mkdir -p ${destdir}${docdir}/numactl/examples ; \
+		install -m 0644 numademo.c ${destdir}${docdir}/numactl/examples ; \
 	fi	
 
 HTML := html/numactl.html html/numa.html
