--- tuxguitar		2009-11-01 17:25:02.000000000 +0400
+++ tuxguitar.new	2011-01-11 10:26:55.000000000 +0300
@@ -17,13 +17,13 @@
 fi
 ##LIBRARY_PATH
 LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:lib/
-LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/lib
-LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/lib
-LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/lib/jni
-LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
+LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/lib64
+LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/lib64
+LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/lib64/jni
+LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib64
 LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${MOZILLA_FIVE_HOME}
 ##CLASSPATH
-CLASSPATH=$CLASSPATH:./tuxguitar.jar:lib/swt.jar:lib/itext.jar:share/
+CLASSPATH=$CLASSPATH:/usr/share/tuxguitar/tuxguitar.jar:/usr/share/tuxguitar/swt.jar:/usr/share/tuxguitar/itext.jar:/usr/share/tuxguitar/
 ##MAINCLASS
 MAINCLASS=org.herac.tuxguitar.gui.TGMain
 ##JVM ARGUMENTS
@@ -33,4 +33,4 @@
 export LD_LIBRARY_PATH
 export MOZILLA_FIVE_HOME
 ##LAUNCH
-${JAVA} ${VM_ARGS} -cp :${CLASSPATH} -Dtuxguitar.share.path="share/" -Djava.library.path="${LD_LIBRARY_PATH}" ${MAINCLASS} "$1" "$2"
+${JAVA} ${VM_ARGS} -cp :${CLASSPATH} -Dtuxguitar.share.path="/usr/share/tuxguitar/" -Djava.library.path="${LD_LIBRARY_PATH}" ${MAINCLASS} "$1" "$2"
