--- inc/Module/Install/PRIVATE/Net/SSLeay.pm_old	2012-09-25 01:08:58.000000000 +0300
+++ inc/Module/Install/PRIVATE/Net/SSLeay.pm	2012-11-10 15:10:43.890000001 +0300
@@ -47,15 +47,15 @@
         LIBS => join(' ', (map {"-L$_"} @{$opts->{lib_paths}}), (map {"-l$_"} @{$opts->{lib_links}})),
     );
 
-    if ( $self->prompt(
-            "Do you want to run external tests?\n".
-            "These tests *will* *fail* if you do not have network connectivity.",
-            'n',
-    ) =~ /^y/i ) {
-        $self->tests('t/*/*.t t/*/*/*.t');
-    } else {
+#    if ( $self->prompt(
+#            "Do you want to run external tests?\n".
+#            "These tests *will* *fail* if you do not have network connectivity.",
+#            'n',
+#    ) =~ /^y/i ) {
+#        $self->tests('t/*/*.t t/*/*/*.t');
+#    } else {
         $self->tests('t/local/*.t t/handle/local/*.t');
-    }
+#    }
 }
 
 sub ssleay_get_build_opts {
