*** x11vnc_loop	2011-06-19 16:22:05.498000019 +0300
--- x11vnc-0.9.12/x11vnc/misc/x11vnc_loop	2005-04-09 18:41:11.000000000 +0300
***************
*** 14,26 ****
  ##########################################################################
  # The following needs to be customized:
  x11vnc_cmd=x11vnc		# or use full path (or set PATH).
! pwfile=/etc/x11vnc/passwd	# always use a password
  display=:0			# display of interest
  restart_sleep=5 		# pause between X server restarts.
  
  # modify cmdline args if desired:
! #x11vnc_args="-display $display -rfbauth $pwfile -forever -nap"
! x11vnc_args="-display $display -rfbauth $pwfile -forever -nap -ncache 10 -ncache_cr"
  
  # you may need to customize the "grep", etc, below in get_xauthority_file()
  ##########################################################################
--- 14,25 ----
  ##########################################################################
  # The following needs to be customized:
  x11vnc_cmd=x11vnc		# or use full path (or set PATH).
! pwfile=/path/to/vnc/passwd	# always use a password
  display=:0			# display of interest
  restart_sleep=5 		# pause between X server restarts.
  
  # modify cmdline args if desired:
! x11vnc_args="-display $display -rfbauth $pwfile -forever -nap"
  
  # you may need to customize the "grep", etc, below in get_xauthority_file()
  ##########################################################################
***************
*** 46,52 ****
  	## echo `ls -t /var/lib/xdm/authdir/authfiles/* | head -1`; return
  
  	# this hack tries to grep it out of ps output...
! 	xauth="ls -t /var/gdm/*auth* | head -1"
  	for i in 1 2 3
  	do
  		# very linux specific, and you likely need to tweak..
--- 45,51 ----
  	## echo `ls -t /var/lib/xdm/authdir/authfiles/* | head -1`; return
  
  	# this hack tries to grep it out of ps output...
! 	xauth=""
  	for i in 1 2 3
  	do
  		# very linux specific, and you likely need to tweak..
