diff -uNr qt-gstreamer-0.10.1.orig/elements/CMakeLists.txt qt-gstreamer-0.10.1/elements/CMakeLists.txt
--- qt-gstreamer-0.10.1.orig/elements/CMakeLists.txt	2011-01-23 12:35:19.000000000 +0000
+++ qt-gstreamer-0.10.1/elements/CMakeLists.txt	2011-04-14 16:57:43.000000000 +0000
@@ -15,5 +15,5 @@
     target_link_libraries(gstqwidgetvideosink ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY}
                                               ${GOBJECT_LIBRARIES} ${GSTREAMER_LIBRARY}
                                               ${GSTREAMER_BASE_LIBRARY} ${GSTREAMER_VIDEO_LIBRARY})
-    install(TARGETS gstqwidgetvideosink DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/gstreamer-0.10)
+    install(TARGETS gstqwidgetvideosink DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/gstreamer-0.10)
 endif()
diff -uNr qt-gstreamer-0.10.1.orig/src/CMakeLists.txt qt-gstreamer-0.10.1/src/CMakeLists.txt
--- qt-gstreamer-0.10.1.orig/src/CMakeLists.txt	2011-01-23 12:35:19.000000000 +0000
+++ qt-gstreamer-0.10.1/src/CMakeLists.txt	2011-04-14 16:48:29.000000000 +0000
@@ -1,5 +1,5 @@
 # Common variables
-set(LIB_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib)
+set(LIB_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 set(INCLUDES_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/include/QtGStreamer)
 set(EXPORT_TARGET_SET QtGStreamerTargets)
 
