--- lsb_release.orig	2000-12-15 10:19:16.000000000 +0300
+++ lsb_release	2012-09-09 20:05:32.455974064 +0400
@@ -4,7 +4,12 @@
 # Licence: GPL (latest version), Free Software Group, Inc
 # Author:  Dominique MASSONIE <mdomi@users.sourceforge.net>
 # Date:    September 27th, 2000
+# Patched: Valdos <fat0troll> Sinus <fat0troll@riseup.net> for AgiliaLinux
+# Date:    December 31th, 2010
 #
+# * Changes from patch 1.4.1 (non-upstream)
+# - removed Debian-related strings as unused in AgiliaLinux
+# - added strings for AgiliaLinux version (/etc/agilialinux-version)
 # * Changes in 1.4
 # - "awk" not needed anymore (Loic Lefort)
 # - fixed bug #121879 reported by Chris D. Faulhaber,
@@ -44,9 +49,11 @@
 # Defines the data files
 INFO_ROOT="/etc"                              # directory of config files
 INFO_LSB_FILE="lsb-release"                   # where to get LSB version
-INFO_DISTRIB_SUFFIX="release"                 # <distrib>-<suffix>
-ALTERNATE_DISTRIB_FILE="/etc/debian_version"  # for Debian [based distrib]
-ALTERNATE_DISTRIB_NAME="Debian"               #     "
+# Here is changed for Agilia ;) /etc/agilialinux-version
+INFO_DISTRIB_SUFFIX="version"                 # <distrib>-<suffix>
+# Here was items for Debian-based distros, useless in Agilia
+#ALTERNATE_DISTRIB_FILE="/etc/debian_version"  # for Debian [based distrib]
+#ALTERNATE_DISTRIB_NAME="Debian"               #     "
 CHECKFIRST="/etc/redhat-release"              # check it before file search
 
 # Defines our exit codes
@@ -179,30 +186,13 @@
 
     if [ -z "$FILENAME" ]
     then
-        if [ -f "$ALTERNATE_DISTRIB_FILE" ]
-        then # For Debian only
-            [ -z "$DISTRIB_ID" ] && DISTRIB_ID="$ALTERNATE_DISTRIB_NAME"
-            [ -z "$DISTRIB_RELEASE" ]                                         \
-                && DISTRIB_RELEASE=$(cat $ALTERNATE_DISTRIB_FILE)
-            [ -z "$DISTRIB_CODENAME" ] && [ "$DISTRIB_RELEASE" = "2.1" ]      \
-                && DISTRIB_CODENAME="Slink"
-            [ -z "$DISTRIB_CODENAME" ] && [ "$DISTRIB_RELEASE" = "2.2" ]      \
-                && DISTRIB_CODENAME="Potato"
-#           [ -z "$DISTRIB_CODENAME" ] && [ "$DISTRIB_RELEASE" = "2.3" ]      \
-#               && DISTRIB_CODENAME="Woody"
-            [ -z "$DISTRIB_CODENAME" ] && DISTRIB_CODENAME=$DISTRIB_RELEASE
-            # build the DISTRIB_DESCRIPTION string (never need to be parsed)
-            [ -z "$DISTRIB_DESCRIPTION" ]                                     \
-                && DISTRIB_DESCRIPTION="$DISTRIB_ID $DESCSTR_DELI $DISTRIB_REL\
-EASE ($DISTRIB_CODENAME)"
-        else # Only for nothing known compliant distrib :(
+        # Only for nothing known compliant distrib :(
             [ -z "$DISTRIB_ID" ] && DISTRIB_ID=$MSG_NA
             [ -z "$DISTRIB_RELEASE" ] && DISTRIB_RELEASE=$MSG_NA
             [ -z "$DISTRIB_CODENAME" ] && DISTRIB_CODENAME=$MSG_NA
             [ -z "$DISTRIB_DESCRIPTION" ] && DISTRIB_DESCRIPTION=$MSG_NONE
 
             EXIT_STATUS=$ERROR_NOANSWER
-        fi
     else
         NO=""                    # is Description string syntax correct ?
         if [ -z "$DISTRIB_DESCRIPTION" ]                                      \
